yaksh/templates/yaksh/add_quiz.html

```diff
@@ -37,8 +37,17 @@
 {% if quiz and course_id %}
     {% if quiz.questionpaper_set.get.id %}
     <center>
-    <h4>You can check the quiz by attempting it in the following modes:</h4>
       <a href="{{URL_ROOT}}/exam/manage/designquestionpaper/{{ quiz.id }}/{{quiz.questionpaper_set.get.id}}/{{course_id}}" class="btn btn-primary">View Question Paper</a>
+      {% if user.profile.is_moderator %}
```

Comment:
Why is this check needed here!
Only a moderator can view edit quiz, so it's already checked.
And in the preview questionpaper view this check is present.
Also, its always better to have such check in the view.

Reply:
Removed this.